### PR TITLE
feat: tags support and fix changelogs 

### PR DESCRIPTION
**Pull request by indrastorms**

### CODE REVIEW

**Comment by alexandreteles** [comment]
Under review.

File: app/controllers/Releases.py
Comment (alexandreteles) [inline app/controllers/Releases.py:1]: Is it possible to rethink the usage of regular expressions and use string substitutions and/or list comprehensions/for/foreach? This: https://thedeveloperblog.com/c-sharp/regex-versus-loop is true for Python, and most of the API code was written thinking on optimizing for speed.

Comment (Indranil012) [reply app/controllers/Releases.py:1]: > 

i will try to do that, if i couldn't do it will revert regex and output commit without cleanup

**Comment by alexandreteles** [comment]
Running the workflow again so I can finally start review and test again.

**Review by alexandreteles — APPROVED** [review]
Approved for the rewrite.

**Pull request merged**